File: 백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.py
import sys

input =__import__('sys').stdin.readline

s,k,h = map(int,input().split())

if s + k + h >= 100 :
    print("OK")
elif s + k + h < 100 :
    if s > k :
        if k > h :
            print("Hanyang") # S K H
        elif k < h :
            print("Korea") # S H K 
    elif s < k :
        if s < h :
            print("Soongsil") # K H S
        elif s > h :
            print("Hanyang") # K S H
    elif s < h :
        if s < k :
            print("Soongsil") # H K S
        elif s > k :
            print("Korea") # H S K

File: 백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.c

int main()
{
    int S,K,H;
    int min,max;
    scanf("%d %d %d", &S, &K, &H);
    
    int sum = S + K + H;
    if(sum >= 100)
    {
        printf("OK");
    }
    else if(sum < 100)
    {
        if(S > K)
        {
            max = S;
            min = K;
        }
        else
        {
            max = K;
            min = S;
        }
        if(H > max)
        {
            max = H;
        }
        if(H < min)
        {
            min = H;
        }
        if(min == S)
        {
            printf("Soongsil");
        }
        else if(min == K)
        {
            printf("Korea");
        }
        else if(min == H)
        {
            printf("Hanyang");
        }
    }
}